modules/phase_field/include/postprocessors/FeatureFloodCount.h

```diff
@@ -91,18 +91,13 @@ class FeatureFloodCount :
     {
     }
 
-    FeatureData(const FeatureData & f) :
-        _ghosted_ids(f._ghosted_ids),
-        _local_ids(f._local_ids),
-        _halo_ids(f._halo_ids),
-        _periodic_nodes(f._periodic_nodes),
-        _var_idx(f._var_idx),
-        _bboxes(f._bboxes),
-        _min_entity_id(f._min_entity_id),
-        _status(NOT_MARKED),
```

Comment:
OK, this is fine. I don't do any copying at all now (to my knowledge), and even if I do the `_status` variable wouldn't be expected to be preserved or used at all. In fact, I'm somewhat tempted to `delete` both the copy constructor and copy assignment operator.... I wonder if this is good practice?